misc/cleanup-dash.py

```diff
@@ -194,14 +212,57 @@ def add_annotation(dashboard):
             dashboard['annotations']['list'].append(add_item)
     return dashboard
 
+def add_copyrights_links(dashboard):
+    """Add Copyrights Links footer."""
+
+    prompt = 'Add Copyrights Links - "Obligate for pmmdemo" (conventional: **No**) [%s]: ' % (
+        "No",
+    )
+    user_input = raw_input(prompt)
+    if user_input:
+        if user_input == 'Yes':
+            add_item = {
+                'collapsed': False,
+                'gridPos': {
+                    'h': 1,
+                    'w': 24,
+                    'x': 0,
+                    'y': 99
+                },
+                'id': 9998,
+                'panels': [],
+                'title': "Copyrights & Legal",
+                'type': "row"
+            }
+            dashboard['panels'].append(add_item)
+            add_item = {
+                'content': "<center >\n  <p>MySQL and InnoDB are trademarks of Oracle Corp. Proudly running Percona Server. Copyright (c) 2006-2018 Percona LLC.</p>\n  <div style=\"text-align:center;\">\n    <a href=\"https://percona.com/terms-use\" style=\"display: inline;\">Terms of Use</a> | \n    <a href=\"https://percona.com/privacy-policy\" style=\"display: inline;\">Privacy</a> | \n    <a href=\"https://percona.com/copyright-policy\" style=\"display: inline;\">Copyright</a> | \n    <a href=\"https://percona.com/legal\" style=\"display: inline;\">Legal</a>\n  </div>\n</center>\n<hr>\n<link rel=\"stylesheet\" type=\"text/css\" href=\"//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css\" />\n<script src=\"//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js\"></script>\n<script>\nfunction bbb(){\n  \n  setTimeout(function (){ \n  window.cookieconsent.initialise({\n    \"palette\": {\n      \"popup\": {\n        \"background\": \"#eb6c44\",\n        \"text\": \"#ffffff\"\n      },\n      \"button\": {\n        \"background\": \"#f5d948\"\n      }\n    },\n    \"theme\": \"classic\",\n    \"content\": {\n      \"message\": \"This site uses cookies and other tracking technologies to assist with navigation, analyze your use of our products and services, assist with promotional and marketing efforts, allow you to give feedback, and provide content from third parties. If you do not want to accept cookies, adjust your browser settings to deny cookies or exit this site.\",\n      \"dismiss\": \"Allow cookies\",\n      \"link\": \"Cookie Policy\",\n      \"href\": \"https://www.percona.com/cookie-policy\"\n    }\n  })},3000)};\n  \n  \n  window.addEventListener(\"load\",bbb());\n\n\n\n</script>",
```

Comment:
Please, use single quotes here, and no escapes of double quotes in html  :)

Reply:
Thx, quotes have been replaced.